src/components/Notifications/hooks/use-notification-list.hook.tsx

```diff
@@ -370,7 +372,9 @@ export const useNotificationList = (
               </div>
             ),
             createdAt: notification.createdAt,
-            href: `support@myriad.social`,
+            href: notification.message.includes('approved')
+              ? ''
+              : `support@myriad.social?subject=Complain post take down!`,
```

Comment:
please use env config to get mail support

https://github.com/myriadsocial/myriad-web/blob/12d02c06683f46f72306ed2f211664b6394a0c46/next.config.js#L36